astropy/cosmology/flrw.py

```diff
@@ -153,7 +153,7 @@ def __init__(self, H0, Om0, Ode0, Tcmb0=0.0*u.K, Neff=3.04, m_nu=0.0*u.eV,
         cd0value = critdens_const * H0_s ** 2
         self._critical_density0 = u.Quantity(cd0value, u.g / u.cm ** 3)
 
-        # Load up neutrino masses.  Note: in Py2.x, floor is floating
+        # Load up neutrino masses.
         self._nneutrinos = int(floor(self._Neff))
```

Comment:
Given the comment, it would seem we could remove the `int(...)` part of this.

Reply:
Done! Python 3 floor is an int. Just checked.